controls/5_2_ssh_server_configuration.rb

```diff
@@ -33,168 +33,221 @@
   impact 1.0
 
   tag cis: 'distribution-independent-linux:5.2.10'
   tag level: 1
 
   describe sshd_config do
-    its(:PermitUserEnvironment) { should eq 'no' }
+    its('PermitRootLogin') { should eq 'no' }
   end
 end
 
 control 'cis-dil-benchmark-5.2.11' do
-  title 'Ensure only approved ciphers are used'
-  desc "This variable limits the types of ciphers that SSH can use during communication.\n\nRationale: Based on research conducted at various institutions, it was determined that the symmetric portion of the SSH Transport Protocol (as described in RFC 4253) has security weaknesses that allowed recovery of up to 32 bits of plaintext from a block of ciphertext that was encrypted with the Cipher Block Chaining (CBD) method. From that research, new Counter mode algorithms (as described in RFC4344) were designed that are not vulnerable to these types of attacks and these algorithms are now recommended for standard use."
+  title 'Ensure SSH PermitEmptyPasswords is disabled (Scored)'
+  desc  "The PermitEmptyPasswords parameter specifies if the SSH server allows login to accounts with empty password strings.\n\nRationale: Disallowing remote shell access to accounts that have an empty password reduces the probability of unauthorized access to the system"
   impact 1.0
 
   tag cis: 'distribution-independent-linux:5.2.11'
   tag level: 1
 
   describe sshd_config do
-    its(:Ciphers) { should_not be_nil }
+    its('PermitEmptyPasswords') { should eq 'no' }
+  end
+end
+
+control 'cis-dil-benchmark-5.2.12' do
+  title 'Ensure SSH PermitUserEnvironment is disabled (Scored)'
+  desc  "The PermitUserEnvironment option allows users to present environment options to the ssh daemon.\n\nRationale: Permitting users the ability to set environment variables through the SSH daemon could potentially allow users to bypass security controls (e.g. setting an execution path that has ssh executing trojan'd programs)"
+  impact 1.0
+
+  tag cis: 'distribution-independent-linux:5.2.12'
+  tag level: 1
+
+  describe sshd_config do
+    its('PermitUserEnvironment') { should eq 'no' }
+  end
+end
+
+control 'cis-dil-benchmark-5.2.13' do
+  title 'Ensure only strong Ciphers are used (Scored)'
+  desc "This variable limits the types of ciphers that SSH can use during communication.\n\nRationale: Weak ciphers that are used for authentication to the cryptographic module cannot be relied upon to provide confidentiality or integrity, and system data may be compromised"
+  impact 1.0
+
+  tag cis: 'distribution-independent-linux:5.2.13'
+  tag level: 1
+
+  describe sshd_config do
+    its('Ciphers') { should_not be_nil }
   end
 
+  WEAK_CIPHERS = [
```

Comment:
Would it be better to implement like this?
https://github.com/Logicworks/cis-dil-benchmark/blob/74a01948ac862a5d46c82587b9dff29b37104674/controls/3_2_network_parameters_host_and_router.rb#L49-L59

Reply:
Changed 'desc' formatting to match the suggested. Thanks